assets/src/settings-page/settings-footer.js

```diff
@@ -78,7 +85,20 @@ export function SettingsFooter() {
 						type={ NOTICE_TYPE_SUCCESS }
 					>
 						<p>
-							{ __( 'Saved', 'amp' ) }
+							{
+								SITE_HAS_CACHE_ENABLE && shouldShowPageCacheFlushNotice ? (
+									<>
+										{ __( 'Mode saved. Please flush page cache. ', 'amp' ) }
```

Comment:
Probably best to not rely on translators to retain the trailing space. Also, since other settings may have been modified, probably best to not limit to “Mode”.

```suggestion
										{ __( 'Saved. Consider flushing page cache.', 'amp' ) + ' ' }
```